### ISSUE TITLE
How to open one file in two columns?

### ISSUE THREAD

**Issue by losymear** [tamlok/vnote#269]
Wonder if it's possible to open one file in two columns, one in **edit mode** and the other in **read mode**, thus I could see the rendered view instantly when I'm editing.

**Comment by tamlok** [comment]
No, VNote does not prefer the side-by-side live preview for the whole document. It is sufficient to just focus in edit mode with the help of syntax highlight and in-place preview. For diagram, you could use `Ctrl+E I` to open a live preview panel for a diagram only.

Thanks!
